core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/ApplicationModuleFinder.java

```diff
@@ -185,5 +191,25 @@ protected void addDependencies(ModuleSpec.Builder builder, ApplicationEnvironmen
                 });
     }
 
+    private void addJDBCDrivers(ModuleSpec.Builder builder) throws IOException {
```

Comment:
I understand the primary motivation is JDBC drivers, but it's actually pretty universal. The name is slightly misleading :-)

Reply:
That's when that was the original purpose, but it became more general purpose than JDBC. Will fix the name